### ISSUE TITLE
Unexpected input(s) 'audience'

### ISSUE THREAD

**Issue by fabiendelpierre** [aws-actions/configure-aws-credentials#475]
Hello,

I have the following step:
```
      - name: Configure AWS Credentials
        uses: aws-actions/configure-aws-credentials@v1
        with:
          aws-region: ${{ env.TF_VAR_aws_region }}
          audience: custom-audience-string
          role-to-assume: ${{ secrets.ROLE_ARN }}
          role-session-name: ${{ env.GITHUB_SHA }}
```

With the `audience` line, the job returns:

Warning: Unexpected input(s) 'audience', valid inputs are ['aws-access-key-id', 'aws-secret-access-key', 'aws-session-token', 'aws-region', 'mask-aws-account-id', 'role-to-assume', 'web-identity-token-file', 'role-duration-seconds', 'role-session-name', 'role-external-id', 'role-skip-session-tagging']

My IAM OIDC provider is configured to accept the `custom-audience-string` but it looks like it's not even making it there

Everything works fine if I omit the `audience` argument, but I was hoping to use one OIDC provider for multiple workflows using multiple audiences. I don't understand why I'm getting this error when `audience` is both documented and present in the code.

Probably missing something silly because it's Friday afternoon but... please point me to it? Thank you :)

**Comment by austinvalle** [comment]
Just ran into this, looks like the PR was merged to support this #362 , however there hasn't been a new release created yet: https://github.com/aws-actions/configure-aws-credentials/releases

Until the release is created, you can reference the action via `master` branch or [commit ](https://github.com/aws-actions/configure-aws-credentials/tree/34779ed730da18be6e68b2bd0122feff57a542e5) like below:
```yml
      - name: configure aws credentials
        uses: aws-actions/configure-aws-credentials@34779ed730da18be6e68b2bd0122feff57a542e5

      - name: configure aws credentials
        uses: aws-actions/configure-aws-credentials@master
```

**Comment by fabiendelpierre** [comment]
Aah... I has looked through the code to make sure the audience was in there, but of course I didn't check the dates on the commits and didn't realize it was so recent. Thank you!
